generate/beat/{{cookiecutter.beat}}/dev-tools/set_version

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```

Comment:
We should find a way to use the generic set version script and pass extra params. Otherwise we run into the same problem we had before that we update a script and people update libbeat but don't get the newest scripts.